apps/coscrad-frontend/src/components/resources/bibliographic-references/book-bibliographic-reference-detail.full-view.presenter.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+    IBibliographicReferenceViewModel,
+    IBookBibliographicReferenceData,
+    IValueAndDisplay,
+} from '@coscrad/api-interfaces';
+import { Card, CardContent, CardHeader, Divider } from '@mui/material';
+import { SinglePropertyPresenter } from '../../../utils/generic-components';
+
+export const BookBibliographicReferenceDetailFullViewPresenter = ({
+    data,
+}: IBibliographicReferenceViewModel<IBookBibliographicReferenceData>): JSX.Element => {
+    const { title, abstract, year, publisher, place, url, numberOfPages, isbn } = data;
+
+    const labelsAndValues: IValueAndDisplay<unknown>[] = (
+        [
+            [abstract, 'Abstract'],
+            [year, 'Year'],
+            [publisher, 'Publisher'],
+            [place, 'Place'],
+            // TODO format this as a link
+            [url, 'External Link'],
+            [numberOfPages, 'Page Count'],
+            [isbn, 'ISBN'],
+            // TODO Expose creators
+        ] as const
+    )
+        // Do not present optional values
```

Comment:
extra indent?

Reply:
Nicely implemented

Reply:
No that's Prettier's formatting. The comment applies to the chained callback, and should be at the same level of indentation as the '.' preceding the `filter` call.